packages/govern-console/src/containers/DAO/DaoHomePage.tsx

```diff
@@ -1,28 +1,151 @@
+import styled from 'styled-components';
 import { ApmRoute } from '@elastic/apm-rum-react';
-import { Redirect, Switch, useRouteMatch } from 'react-router';
+import { Grid, GridItem, GU, useLayout } from '@aragon/ui';
+import { lazy, Suspense, useEffect, useState } from 'react';
+import { Redirect, Switch, useHistory, useLocation, useParams, useRouteMatch } from 'react-router';
 
 import DaoSideCard from './components/DaoSideCard/DaoSideCard';
-import DaoSettings from 'containers/DAOSettings/DAOSettings';
+import NewExecution from 'containers/NewExecution/NewExecution';
+import HelpComponent from 'components/HelpComponent/HelpComponent';
+import ProposalDetails from 'containers/ProposalDetails/ProposalDetails';
+import { useDaoQuery, useLazyProposalListQuery } from 'hooks/query-hooks';
 
+const DaoSettings = lazy(() => import('containers/DAOSettings/DAOSettings'));
+
+const StyledGridItem = styled(GridItem)<{ layoutName: string }>`
+  padding-top: ${(props) => (props.layoutName === 'small' ? undefined : 3 * GU)}px;
+`;
+
+/**
+ * Main page taking care of the routing to various dao functions;
+ * pulls all the data (for now at least)
+ */
 const DaoHomePage: React.FC = () => {
+  const history = useHistory();
+  const { daoName } = useParams<any>();
+  const { pathname } = useLocation();
   const { path, url } = useRouteMatch();
+  const { layoutName } = useLayout();
+
+  /**
+   * State
+   */
+  const [daoExists, setDaoExists] = useState<boolean>(true);
+  const [daoDetails, setDaoDetails] = useState<any>();
```

Comment:
I think this is not needed (indeed, the `daoDetails` is unused). You're already getting that info from `useDaoQuery`. Is there a good reason for duplicating it?